### PR TITLE
Added Schroeder-2012-824 norm data

### PR DESCRIPTION
**Pull request by AnnikaTjuka**
**About the data:**

German norms for semantic typicality, age of acquisition, and concept familiarity for 824 exemplars of 11 semantic categories, including four natural (animals, birds, fruits, and vegetables) and five man-made (clothing, furniture, vehicles, tools, and musical instruments) categories, as well as professions and sports. Each category exemplar in the data was collected empirically in an exemplar generation study.

**About the mapping:**
- [ ]  I did not map all sub terms to the according key term, for example: _Schwarzbär_, _Braunbär_ are not mapped to BEAR. There are many instances of sub terms and I didn't want to use the same concept several times, e.g. _Schreibtisch_, _Couchtisch_, _Beistelltisch_ etc. Therefore, I tried to only map the key terms, _Tisch_ = TABLE.
- [ ]  If there are synonyms, e.g., _Nilpferd_ and _Flusspferd_, I mapped both to HIPPOPOTAMUS.
- [ ]  Also some of the words occur twice in the semantic categories "animals" **and** "birds". 
- [ ]  The following verbs are listed as sports: _Fishing_, _Flying_, _Swimming_. Can we map them to the according verbs?

**About the metadata:**
- In the columns with frequency values, some rows have the value `None`. Therefore, I couldn't use `float` as a datatype. Is this a problem?

**New concepts:**
- [ ]  I added new concepts but wasn't sure which semantic field to use for the vegetables, e.g., _brokkoli_, _zucchini_ etc. In the concepticon list some vegetables are listed as "Agriculture and vegetation": see 2365 RADISH
- [ ]  The list includes many new concepts in the categories clothing, vehicle, instruments, sports, and profession. We might want to add those in the future. Should I add the list to the left-over concepts issue: https://github.com/concepticon/pyconcepticon/issues/20 ?

### CODE REVIEW

File: concepticondata/conceptlists/Schroeder-2012-824.tsv [hunk 1]
```diff
@@ -0,0 +1,825 @@
...
+Schroeder-2012-824-16	16	Kamel	camel	CAMEL	1871	animal	1.9	1.2096106376586	2.55	0.825577947481896	2.75	1.1180339887499	2.73047638466	0.436238424763	5	2
+Schroeder-2012-824-17	17	Feuerkäfer	Cardinal beetle			animal	4.875	1.7464249196573	4.57894736842105	1.50243467129871	1.70588235294118	0.985184366143778	0.00817507899599	-2.08750804205	7	4
+Schroeder-2012-824-18	18	Katze	cat	CAT	1208	animal	1.1	0.447213595499958	1.35	0.489360484929593	4.1	1.25236618152663	19.3095365885	1.28577185123	4	2
+Schroeder-2012-824-19	19	Chamäleon	chameleon	CHAMELEON	1555	animal	3.3	1.8093325317714	4.45	1.27630222456166	1.9	0.911909506128991	0.686706635663	-0.163228755987	8	4
+Schroeder-2012-824-20	20	Gepard	cheetah	CHEETAH	3146	animal	2.8	1.82381901225798	4.35	1.38696943388321	2.47368421052632	1.34859684498089	0.2043769749	-0.689568033377	6	2
+Schroeder-2012-824-21	21	Huhn	chicken	CHICKEN	1318	animal	1.8	1.64156536333625	1.6	0.502624689950034	4.5	0.688247201611685	11.9029150182	1.07565333293	3	1
+Schroeder-2012-824-22	22	Schimpanse	chimpanzee	APE OR MONKEY	3298	animal	2.3	1.71985311490316	3.85	1.42441123571146	3.2	1.15165784392487	4.71702058069	0.673667771107	8	3
```
Comment (Schweikhard) [inline concepticondata/conceptlists/Schroeder-2012-824.tsv]: I would leave this unmapped or add it to a new category.

**Comment by AnnikaTjuka** [comment]
Thanks for the comments! 

> Frettchen ferret WEASEL 3384
> A Frettchen is a domesticated polecat (Iltis), a different species from the weasel.
>
I mapped _ferret_ to WEASEL because our concepticon description is as follows:
"A small predator of the genus Mustela characterized by a slender body that lets it hunt burrowing animals."
And in Wiktionary, I found the following description of _ferret_:
"An often domesticated mammal (Mustela putorius furo) rather like a weasel, descended from the polecat and often trained to hunt burrowing animals."
I think they all belong to the same species but I will leave it unmapped to avoid confusions.

> Unterhemd vest VEST 3140
> I don't think a vest is an Unterhemd.
>
The concepticon description for VEST is rather ambiguous:
"A piece of clothing for the upper body without sleeves and usually worn together with a shirt."
In Canadian English VEST refers to something that is worn over a shirt, whereas in British English a VEST is worn under a shirt (see [vest in Wiktionary](https://en.wiktionary.org/wiki/vest)).
I will change the description in Concepticon to "**over** a shirt".

**Comment by AnnikaTjuka** [comment]
@Schweikhard I integrated your suggestions in the last commit.

**Comment by AnnikaTjuka** [comment]
@LinguList Before I merge this PR, I just wanted to make sure the `string` instead of `float` in the metadata.json isn't an issue, see changes in file [concepticondata/conceptlists/Schroeder-2012-824.tsv-metadata.json](https://github.com/concepticon/concepticon-data/blob/bca613562bea1a502c9ba032193ae2f3042c5610/concepticondata/conceptlists/Schroeder-2012-824.tsv-metadata.json)
Because some of the values in the frequency columns (FREQ_DLEX and FREQ_LOG10_DLEX) have the value `None`, I couldn't use the datatype `float` in the metadata.json.

**Comment by LinguList** [comment]
@xrotwang or @chrzyki, I know there's a way to specify NULL values in csvw, but I do not know, how one does this, can you quickly help here, so @AnnikaTjuka can use "float" as datatype and specify exceptions for null values?

**Comment by chrzyki** [comment]
Should be doable with the `null` property:

```json
{
  "name": "FREQ_DLEX",
  "null": "None",
  "datatype": {
    "base": "float"
  }
}
```

If I recall correctly.

**Comment by xrotwang** [comment]
yes. and "null" can be a list, too, e.g. ["None", "NA"]

**Comment by AnnikaTjuka** [comment]
Thank you @LinguList, @chrzyki and @xrotwang! It worked.

**Pull request merged**